pgs_line.sql.in

```diff
@@ -593,3 +593,53 @@ COMMENT ON FUNCTION scircle_contains_line_com_neg(sline, scircle) IS
   'returns true if spherical circle does not contain spherical line';
 
 
+--
+-- distance between line and point
+--
+
+CREATE FUNCTION dist(sline, spoint)
+   RETURNS FLOAT8
+   AS 'MODULE_PATHNAME', 'sphereline_point_distance'
+   LANGUAGE 'c'
+   IMMUTABLE STRICT;
+
+COMMENT ON FUNCTION dist(sline, spoint) IS
+  'distance between spherical line and spherical point';
```

Comment:
All the other COMMENT ON FUNCTIONs in this file start with "returns..." I know that seems redundant, but I think it's best to fit in with the convention of the other COMMENTs.